dbimport/main.R

```diff
@@ -55,6 +55,8 @@ fail_if(length(core_files) > 0,
 # import non-core
 ncore_dir <- file.path(getOption("noas")$TABDIR, "non_core")
 table_ids <- list.dirs(ncore_dir, recursive = FALSE, full.names = FALSE)
+if(getOption("noas")$IMPORT_DEBUG == 1) 
```

Comment:
maybe `$IMPORT_DEBUG != "0"` is safer?

Reply:
It could be that all configs are strings by default

Reply:
isnt it safer to have a very explicit debugging flag, and everything else is interpreted as business as usual?

Reply:
yes, I think thats fine too. I was doing two things at the same time. My main point was the quotes

Reply:
`$IMPORT_DEBUG == 1` => `$IMPORT_DEBUG == "1"`

Reply:
oh yeah. quotes sounds good